src/store/actions/healthcheck.actions.js

```diff
@@ -0,0 +1,102 @@
+import R from 'ramda';
```

Comment:
actually, I've added `ramda` here for transforming an object to an array of objects. But I didn't find the easy way to transform( `R.toPairs` is returning an array of arrays, which is not was looking for).